util.go

```diff
@@ -43,15 +44,23 @@ func makeURLValues(v interface{}) url.Values {
 		case reflect.Float64:
 			s = strconv.FormatFloat(fieldValue.Float(), 'f', 4, 64)
 		case reflect.Slice:
-			if field.Type.Elem().Kind() == reflect.Uint8 {
-				s = string(fieldValue.Bytes())
```

Comment:
Was this unused, basically?

Reply:
It seems so. I can't find any endpoints using a byte slice. There are only 2 endpoints with slices in their request structs and they're both string slices.